ipld/blockstore/src/lib.rs

```diff
@@ -37,6 +37,10 @@ pub trait BlockStore: Store {
         Ok(self.read(cid.to_bytes())?)
     }
 
+    fn get_bytes2(&self, cid: &Cid) -> anyhow::Result<Option<Vec<u8>>> {
```

Comment:
Those `name2` names are not very descriptive. Could we get rid of the original `get_bytes` to snatch the name? Alternatively, `get_bytes_anyhow` would provide more context.

Reply:
I believe we have to keep the original names until we've moved completely to FVM actors. I've renamed the functions to `*_anyhow` for now.